tests/build/chains.go

```diff
@@ -120,8 +127,10 @@ var _ = framework.ChainsSuiteDescribe("Tekton Chains E2E tests", func() {
 				Bundle:          framework.TektonController.Bundles.HACBSTemplatesBundle,
 			}
 			pr, waitTrErr := kubeController.RunPipeline(generator, pipelineRunTimeout)
+			g.GinkgoWriter.Printf("Running pipeline %s\n", pr.Name)
 			Expect(waitTrErr).NotTo(HaveOccurred())
 			waitErr := kubeController.WatchPipelineRun(pr.Name, pipelineRunTimeout)
+			g.GinkgoWriter.Printf("Waiting for pipeline %s to finish\n", pr.Name)
```

Comment:
I think this one, should move up because `WatchPipelineRun` may take a while to complete. A message before calling it makes sense to provide some feedback and set user expectations.